auth/src/main/java/com/vimeo/networking2/utils/AuthExtensions.kt

```diff
@@ -0,0 +1,47 @@
+package com.vimeo.networking2.utils
```

Comment:
Should this file utilize `@file:JvmName` for better java interop or are we okay with the name being `AuthExtensions`?

Reply:
This extension will only be used internally by the SDK. Therefore, I didn't add `@file:JvmName`. If public extensions are added to this class later on, then I could create a public extensions file and add `@file:JvmName`.